typo3conf/LocalConfiguration.php

```diff
@@ -9,7 +9,7 @@
     ],
     'DB' => [
         'database' => 't3kit',
-        'host' => 'localhost',
+        'host' => 'db',
```

Comment:
@sakonachhoeurng please test does this affect the functionality when running on Docker. It might not get connected with localhost.

cc: @MattiasNilsson